### PR TITLE
updates header in readme

### PR DESCRIPTION
**Pull request by katiehoesley**
## What?

Write a description of the changes you're making here, and decisions that you made
along the way. Provide enough context that the reviewer(s) can understand your
changes, and the intent behind them. For more details refer to [contributing guideline](https://github.com/bigcommerce/bigcommerce/blob/master/CONTRIBUTING.md#what)

## Why?

Describe the underlying driver or intent behind this change. Provide some context
on why it's needed or what is the intended use. For more details refer to [contributing guideline](https://github.com/bigcommerce/bigcommerce/blob/master/CONTRIBUTING.md#why)

## Testing / Proof

Explain what you did to test and ensure that these changes work the way you expect.
You must provide evidence in the form of unit tests, screenshots, logs, API
payloads or other output to show the expected behaviour. For more details refer to [contributing guideline](https://github.com/bigcommerce/bigcommerce/blob/master/CONTRIBUTING.md#testing--proof)

## How can this change be undone in case of failure?

Provide experiments, feature flags or other mechanisms which used to reduce the risk of changes introduced in this PR. For more details refer to [contributing guideline](https://github.com/bigcommerce/bigcommerce/blob/master/CONTRIBUTING.md#how-can-this-change-be-undone-in-case-of-failure)

ping {suggested reviewers}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+
+By choosing to build on top of our Open Source B2B Buyer Portal, you'll have access to build on our B2B buyer portal application backed by a robust set of SaaS APIs. 
+You can get straight to work building for your unique B2B business cases.
+
+![-----------------------------------------------------](https://storage.googleapis.com/bigcommerce-developers/images/catalyst_readme_hr.png)
+
+<p align="center">
+ <a href="https://www.bigcommerce.com/solutions/b2b-ecommerce-platform/">🚀 B2B Editions</a> •
+ <a href="https://developer.bigcommerce.com/community">🤗 BigCommerce Developer Community</a> •
+ <a href="https://bundleb2b.stoplight.io/">📝 B2B Edition API Reference</a> •
```
Comment (bc-micah) [inline README.md:17]: Update the b2b edition api reference link to a better starting point.

```suggestion
 <a href="https://bundleb2b.stoplight.io/docs/openapi/quick-start">📝 B2B Edition API Reference</a> •
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
-A monorepo frontend application designed for the BigCommerce B2B Edition Buyer portal. It's built using Turborepo, TypeScript, and React.
+
+[![MIT License](https://img.shields.io/github/license/bigcommerce/catalyst)](LICENSE.md)
+[![Lighthouse Report](https://github.com/bigcommerce/catalyst/actions/workflows/lighthouse.yml/badge.svg)](https://github.com/bigcommerce/catalyst/actions/workflows/lighthouse.yml) [![Lint, Typecheck, gql.tada](https://github.com/bigcommerce/catalyst/actions/workflows/basic.yml/badge.svg)](https://github.com/bigcommerce/catalyst/actions/workflows/basic.yml)
```
Comment (bc-micah) [inline README.md:3]:
```suggestion
[![MIT License](https://img.shields.io/github/license/bigcommerce/catalyst)](LICENSE.md)
[![Tests](https://github.com/bigcommerce/b2b-buyer-portal/actions/workflows/tests.yaml/badge.svg)](https://github.com/bigcommerce/b2b-buyer-portal/actions/workflows/tests.yaml)
```

**Review by bc-victor — APPROVED** [review]
🎨

File: README.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
-A monorepo frontend application designed for the BigCommerce B2B Edition Buyer portal. It's built using Turborepo, TypeScript, and React.
+<a href="https://bundleb2b.stoplight.io/docs/openapi/quick-start" target="_blank" rel="noopener norerrer">
+  <img src="https://storage.googleapis.com/bigcommerce-developers/images/B2B-edition-1024x683.jpg" alt="B2B Editions Open Source Buyer Portal" title="B2B Editions Open Source Buyer Portal">
```
Comment (deov31) [inline README.md:2]:
```suggestion
  <img src="https://storage.googleapis.com/bigcommerce-developers/images/B2B-edition-1024x683.jpg" alt="B2B Edition Open Source Buyer Portal" title="B2B Edition Open Source Buyer Portal">
```
🍹

**Review by deov31 — APPROVED** [review]

**Pull request merged**